js/Source/Slider.Modify/Slider.Modify.js

```diff
@@ -75,9 +76,22 @@ Slider.Modify = new Class({
     }, this);
   },
 
+  addMoveClass: function(){
+    if(this.options.moveClass && this.options.moveClassTargets){
+      this.options.moveClassTargets.addClass(this.options.moveClass)
+    }
+  },
```

Comment:
If you were to remove your focus from this particular use case you're working on how would you tackle this in a more generic fashion? For example, right now you are adding a class to an arbitrary group of elements specified in the options. Should you remove the class when movement stops? What if you need to do something other than adding a class? What if you want to transition a property? Create an element? Alter a style? And would you want to undo that (or stop doing it) when movement is complete?

Your implementation here is specific to one screen you're trying to tackle. That kind of use case either belongs in the instantiation of the class and its esoteric DOM (and, in Behavior land, that means in the Behavior), or it should be generic, which is basically what events are for.

I'm not saying you can't ship this. Just that you're basically pushing a function (add a class to a collection of elements) into a class that isn't about that. It's about being a slider.